packages/javascript-sdk/src/fr-recovery-codes/script-parser.ts

```diff
@@ -27,4 +28,24 @@ function parseDisplayRecoveryCodesText(text: string): string[] {
   return recoveryCodes || [];
 }
 
-export { parseDisplayRecoveryCodesText };
+/**
+ *
+ * @param text
+ * @returns string
+ */
+function parseDeviceNameText(text: string): string {
+  /**
+   * We default the device name to 'New Security Key'
+   * If the user has a device name, it will be wrapped in <em> tags
+   * e.g. ` ... <em>My Security Key</em> ... `
+   * We want to remove the <em> tags and just return the device name
+   * e.g. ` ... My Security Key ... `
+   */
+  const displayName =
+    text
+      ?.match(/<em\s*.*>\s*.*<\/em>/g)?.[0]
+      ?.replace('<em>', '')
+      ?.replace('</em>', '') ?? 'New Security Key';
```

Comment:
We'll need to keep an eye on this as this could easily break if someone changes the underlying content string this is parsing.

Reply:
Yeah, I wonder how we can be ahead of this? Seems brittle but its just a mess of a string that we only care about this piece of